sdk/containerregistry/azure-containers-containerregistry/src/main/java/com/azure/containers/containerregistry/implementation/UtilsImpl.java

```diff
@@ -85,7 +85,10 @@ public final class UtilsImpl {
 
     public static final HttpHeaderName DOCKER_DIGEST_HEADER_NAME = HttpHeaderName.fromString("docker-content-digest");
     // TODO (limolkova) should we send index and list too so that we won't need to change the default later on?
-    public static final String SUPPORTED_MANIFEST_TYPES = ManifestMediaType.OCI_MANIFEST + "," + ManifestMediaType.DOCKER_MANIFEST;
+    public static final String SUPPORTED_MANIFEST_TYPES = ManifestMediaType.OCI_MANIFEST
```

Comment:
This seems to change a lot, are there any concerns this could become a runtime issue in the future?

Reply:
It's based on open standards and the feedback from the service team, this may take 5+ years for the default to change. 
Also, it will cause behavior changes for other SDKs, docker and azure cli, so it would not happen without some versioning updates.